carbon-design-system/carbon
# refactor(react): update listbox tests to testing-library

Closes #11625

This PR updates our ListBox tests to use react testing library

#### Changelog

**New**

**Changed**

- [x] Use RTL for testing in `ListBox/__tests__/ListBox-test.js`
- [x] Use RTL for testing in `ListBox/__tests__/ListBoxField-test.js`
- [x] Use RTL for testing in `ListBox/__tests__/ListBoxMenu-test.js`
- [x] Use RTL for testing in `ListBox/__tests__/ListBoxMenuIcon-test.js`
- [x] Use RTL for testing in `ListBox/__tests__/ListBoxMenuItem-test.js`
- [x] Use RTL for testing in `ListBox/__tests__/ListBoxSelection-test.js`

**Removed**

#### Testing / Reviewing

- Verify tests now use testing library and do not use enzyme
- Verify test coverage is roughly equivalent between the old and new tests
